### DISCUSSION TITLE
Emails as elements - allows for custom fields/field layouts

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/commerce#2448] [Ideas]
It would be useful if we could attach custom fields to email messages defined in "commerce settings/emails". 

Usa case :
even if i defined email structure in twig, i would like to have some interface for admin to define things like email intro, footer etc.

**Comment by nfourtythree** [comment]
Hi @piotrpog 

Thank you for the feature request. We will consider this for the future.

For the moment we would recommend using a global set or a single entry with custom fields to allow for custom editable content in your emails.

Thanks!

**Comment by piotrpog** [comment]
@nfourtythree 
Thanks, thats what i do :)
